syft_tensorflow/attributes/attributes.py

```diff
@@ -32,12 +32,11 @@ class TensorFlowAttributes(FrameworkAttributes):
     ALIAS = "tensorflow"
     Tensor = TensorFlowTensor
 
-    def __init__(self, tensorflow: ModuleType, hook: "TensorFlowHook"):
+    def __init__(self, tensorflow: ModuleType, hook: ModuleType):
```

Comment:
This should be changed back to "TensorFlowHook" for the `hook` argument type hint